tools/jbuf/generate_plot.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+if [ ! -f .env ]; then
+    echo ".env is missing. Copy and edit env-template!"
+    exit 1
+fi
+
+export `cat .env`
+
+echo "target: $target"
+echo "netif:  $netif"
+echo "once:   $once"
+
+if ! which jq; then
+    echo "Install jq"
+    exit 1
+fi
+
+trap "./jitter-off.sh; killall -q baresip" EXIT
+
+function gen_datfile() {
+    ph=$1
+    filename=$2
+
+    jqc='.traceEvents[] | select (.ph == "'"${ph}"'") | .args.line'
+    cat jbuf.json | jq -r "${jqc}" > $filename
+}
+
+source ./jitter.sh
+init_jitter $netif
+
+strm="audio"
+for jmin in 0 10 20; do
+    for i in 0 1; do
+        if [ "$i" == "0" ]; then
+            strm="audio"
+        else
+            strm="video"
+        fi
+
+        echo "########### jitter buffer $strm $jmin ###############"
+
+        sed -e "s/${strm}_jitter_buffer_delay\s*[0-9]*\-.*/${strm}_jitter_buffer_delay   ${jmin}-500/" -i ${strm}/config
+        baresip -v -f ${strm} > /tmp/${strm}.log 2>&1 &
+        sleep 1
+        echo "/dial $target" | nc -N localhost 5555
+
+        sleep 3
+
+        enable_jitter
+
+        sleep 5
+
+        disable_jitter
+
+        sleep 25
+
+        echo "/quit" | nc -N localhost 5555
+
+        sleep 1
+
+        gen_datfile "P" jbuf.dat
+        gen_datfile "O" overrun.dat
+        gen_datfile "U" underflow.dat
+        gen_datfile "L" toolate.dat
+        gen_datfile "D" duplicate.dat
+        gen_datfile "S" oosequence.dat
+        gen_datfile "T" lost.dat
```

Comment:
I think it would be nice if generating and testing are separated. So you can manually test easily without test loop.

Reply:
The loop is in order to generate plots for different jbuf min settings.

Reply:
I know, but you can extract generating to a separate function so this can be used to run/perform manually tests and generate plots after this.

Reply:
I guess I understood. Like last commit?

Reply:
Thanks!